test/wasm2js.asserts.js

```diff
@@ -76,7 +70,7 @@ function asmFunc0(global, env, buffer) {
 }
 
 var memasmFunc0 = new ArrayBuffer(65536);
-var retasmFunc0 = asmFunc0({Math,Int8Array,Uint8Array,Int16Array,Uint16Array,Int32Array,Uint32Array,Float32Array,Float64Array,NaN,Infinity}, {abort:function() { throw new Error('abort'); }},memasmFunc0);
+var retasmFunc0 = asmFunc0({}, {abort() { throw new Error('abort'); }},memasmFunc0);
```

Comment:
could we also remove empty global`{}` object from `asmFunc` signature?
Like:
```suggestion
var retasmFunc0 = asmFunc0({abort() { throw new Error('abort'); }},memasmFunc0);
```

Reply:
That removes the first parameter, so the function would need to have the parameter removed as well. Seems complicated for little benefit, to me.